### PR TITLE
Restructure printable auction catalogue

### PR DESCRIPTION
**Pull request by svirvel**
Make the printable auction catalogue more condensed by showing 4 items
on each table row.

### CODE REVIEW

File: layouts/auction/printable-catalogue.html [hunk 1]
```diff
@@ -4,53 +4,59 @@
...
-                        {{ $thumbnail := $image.Fill "180x180" }}
-                        <img class="auction-item-thumbnail" src={{$thumbnail.Permalink}}>
-                      {{ end }}
-                    </td>
-                    <td>
-                      <b>{{ index . 0 }}</b><br>
-                      {{ index . 1 }}
-                    </td>
+    <div class="printable-catalogue">
+      {{ $auctionConfig := .Site.Data.AuctionConfig }}
```
Comment (almbr) [inline layouts/auction/printable-catalogue.html:9]: Kan man få be om kommentarer?

File: layouts/auction/printable-catalogue.html [hunk 2]
```diff
@@ -4,53 +4,59 @@
...
+      {{ if (fileExists "content/auction/auctionItems.csv") }}
+
+        {{ $node := . }}
+
+        {{ range .Site.Menus.auction }}
```
Comment (almbr) [inline layouts/auction/printable-catalogue.html:17]: + Kommentar här

File: layouts/auction/printable-catalogue.html [hunk 3]
```diff
@@ -4,53 +4,59 @@
...
+          {{ $category := .Name }}
+          {{ $category := .Name }}
+          <h1>{{ $category }}</h1>
+          <div class="printable-catalogue-table">
+            <table>
+              <tbody>
+                {{ $data := getCSV "," "content/auction/auctionItems.csv"}}
```
Comment (almbr) [inline layouts/auction/printable-catalogue.html:27]: Kommentar här

File: layouts/auction/printable-catalogue.html [hunk 4]
```diff
@@ -4,53 +4,59 @@
...
+                {{ $node.Scratch.Set "i" 1 }}
+                <tr>
+                {{  range $item := after 1 $data }}
```
Comment (almbr) [inline layouts/auction/printable-catalogue.html:34]: Kommentar här

File: layouts/auction/printable-catalogue.html [hunk 5]
```diff
@@ -4,53 +4,59 @@
...
+                  {{ if (in $category (index $item 3)) }}
+                    {{ if eq (mod ($node.Scratch.Get "i") 6) 0 }}
+                      <tr>
+                    {{ end }}
                     <td>
-                      {{ index . 2 }}
-                    </td>
+                      <div class="printable-catalogue-item">
+                        <div class="printable-catalogue-image">
+                          {{ $image := resources.Get (printf "/auction/images/%s.jpg" (index . 5)) }}
```
Comment (almbr) [inline layouts/auction/printable-catalogue.html:45]: Kommentar här

File: layouts/auction/printable-catalogue.html [hunk 6]
```diff
@@ -4,53 +4,59 @@
...
+                          {{ if $image }}
+                            {{ $thumbnail := $image.Fill "180x180" }}
```
Comment (almbr) [inline layouts/auction/printable-catalogue.html:47]: Kommentar här

File: layouts/auction/printable-catalogue.html [hunk 7]
```diff
@@ -4,53 +4,59 @@
...
+                            <img class="auction-item-thumbnail" src={{$thumbnail.Permalink}}>
+                          {{ end }}
+                        </div>
+                        <div class="printable-catalogue-text">
+                          <b>{{ index $item 0 }}</b><br>
+                          {{ index $item 1 }}<br>
```
Comment (almbr) [inline layouts/auction/printable-catalogue.html:53]: Kommentar här maybe??

File: layouts/auction/printable-catalogue.html [hunk 8]
```diff
@@ -4,53 +4,59 @@
...
+                          Status: {{ index $item 2 }}<br>
+                          ID: {{ index $item 5 }}
+                        </div>
+                      </div>
                     <td>
-                      {{ index . 5 }}
-                    </td>
-                  </tr>
+                      {{ $node.Scratch.Add "i" 1 }}
+                      {{ if eq (mod ($node.Scratch.Get "i") 6) 0 }}
```
Comment (almbr) [inline layouts/auction/printable-catalogue.html:63]: Kommentar här

**Comment by svirvel** [comment]
Restructure the printable auction catalogue by ordering the items in a table with 5 columns. This should make the catalogue more compact and printer friendly.

**Pull request merged**